espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/http/RouteMap.java

```diff
@@ -6,17 +6,21 @@
 import javax.annotation.Nullable;
 
 import fi.iki.elonen.NanoHTTPD.Method;
+import io.appium.espressoserver.lib.handlers.exceptions.DuplicateRouteException;
 
 
 class RouteMap {
 
     private final Map<Method, Map<String, RouteDefinition>> routeMap = new ConcurrentHashMap<>();
 
-    public void addRoute(RouteDefinition route) {
+    public void addRoute(RouteDefinition route) throws DuplicateRouteException {
```

Comment:
now there is no need to add DuplicateRouteException into throws list. IMHO the idea of controlled exception is not the best one Java has :)